lib/internal/blob.js

```diff
@@ -436,6 +414,7 @@ ObjectDefineProperties(Blob.prototype, {
   stream: kEnumerableProperty,
   text: kEnumerableProperty,
   arrayBuffer: kEnumerableProperty,
+  bytes: kEnumerableProperty,
```

Comment:
Shouldn't this be a separate commit marked as fix?

Reply:
it's a webidl change